src/lib.rs

```diff
@@ -170,6 +170,22 @@ extern crate bitflags;
 #[macro_use]
 extern crate objc;
 
+/// Taskbar Progress State
+pub enum TaskbarProgressState {
```

Comment:
```suggestion
pub enum ProgressBarState {
```

Reply:
Conflicts with the struct ProgressBarState
Using ProgressState instead

Reply:
right, just call it `ProgressState` then